jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java

```diff
@@ -98,20 +99,32 @@ public static OsTypes getOs() {
     }
 
     // GET DRIVER
-    private static void setBrowserSizeForMac(WebDriver driver,
-            int width, int height) {
+    public static JFunc1<WebDriver, WebDriver> DRIVER_SETTINGS = driver -> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4958360)